src/Components/StackedRankings/StackedRankings.jsx

```diff
@@ -116,6 +116,27 @@ function StackedRankings({ applications, user }) {
           });
           fetched = await GET.getAllStackingsAPI(user);
         }
+        let reviews = await GET.getUserReviewsAPI(user);
+        reviews.map((review)=>{
+          let averageRating = 0;
+          let numRatings = 0;
+          if (review && review.questionList) {
+            review.questionList.map(item => {
+              if (item.rating > 0) {
+                averageRating += item.rating;
+                numRatings += 1;
+              }
+            });
+          }
+          if (numRatings > 0) {
+            averageRating = averageRating / numRatings;
```

Comment:
Do we need to worry about integer rounding?

Reply:
Nope :)